csp-mixin/signals/gcpvpc.libsonnet

```diff
@@ -0,0 +1,295 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: this.filteringSelector,
+    groupLabels: ['job'],
+    instanceLabels: ['project_id'],
+    aggLevel: 'group',
+    discoveryMetric: {
+      stackdriver: 'stackdriver_networking_googleapis_com_location_networking_googleapis_com_fixed_standard_tier_usage',
+    },
+    signals: {
+      //
+      // Fixed Usage Tier
+      //
+      // available labels: ['bandwidth_policy_id', 'location', 'project_id', 'traffic_source']
+      gcpvpc_fixed_usage_tier_network_egress: {
+        name: 'Fixed standard tier network egress',
+        description: 'Rate in bytes of network egress on the fixed standard tier.',
+        type: 'gauge',
+        unit: 'bps',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_networking_googleapis_com_location_networking_googleapis_com_fixed_standard_tier_usage{%(queriesSelector)s}',
+            aggKeepLabels: ['traffic_source'],
+          },
+        },
+      },
+
+      //
+      // Overview counters
+      //
+      gcpvpc_services_in_use_count: {
+        name: 'Total services in use',
+        description: 'Count of unique service names (including "UNKNOWN") with ingress or egress traffic.',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          stackdriver: {
+            expr: 'count(count by (service_name) (stackdriver_google_service_gce_client_networking_googleapis_com_google_service_response_bytes_count{%(queriesSelector)s} OR stackdriver_google_service_gce_client_networking_googleapis_com_google_service_request_bytes_count{%(queriesSelector)s}))',
```

Comment:
Why aren't you counting the requests for `service_name` as you do for responses?

Reply:
Just an oversight.. Possibly a copy/paste error. I'll make sure they're using the same metric, since that makes more sense.

I'm fairly sure I copy/pasted this from a "total throughput" query, which had both request and response metrics in it, and just failed to fully modify the query for this purpose.

Thanks for catching it!

Reply:
Actually.. No.. I responded to this before looking at it closely enough, and before my coffee.

This query is first counting the number of samples for either requests, or responses, by `service_name`. This results in a list of unique service names. Then counting the number of samples returned by that query, gives me the total number of services in use.

Just using request, or response would likely be adequate, but because the count is aggregating on `service_name`, it's safe to include both as a "just in case" measure.